### PR TITLE
< and > wrong

### PR DESCRIPTION
**Pull request by wstaelens**
< and > were not in code

## Summary

Describe your changes here.

Fixes #Issue_Number (if available)

### CODE REVIEW

File: docs/framework/wcf/feature-details/how-to-authenticate-with-a-user-name-and-password.md [hunk 1]
```diff
@@ -22,13 +22,13 @@ ms.workload:
...
   
- To configure a service to authenticate its clients using Windows Domain username and passwords use the <<!--zz xref:System.ServiceModel.WsHttpBinding --> `xref:System.ServiceModel.WsHttpBinding`> and set its `Security.Mode` property to `Message`. In addition you must specify an X509 certificate that will be used to encrypt the username and password as they are sent from the client to the service.  
+ To configure a service to authenticate its clients using Windows Domain username and passwords use the `<xref:System.ServiceModel.WSHttpBinding%2A>` and set its `Security.Mode` property to `Message`. In addition you must specify an X509 certificate that will be used to encrypt the username and password as they are sent from the client to the service.
```
Comment (JRAlexander) [inline docs/framework/wcf/feature-details/how-to-authenticate-with-a-user-name-and-password.md]: Updated the xref for overloading. This was a good catch!

File: docs/framework/wcf/feature-details/how-to-authenticate-with-a-user-name-and-password.md [hunk 2]
```diff
@@ -22,13 +22,13 @@ ms.workload:
...
   
  On the client, you must prompt the user for the username and password and specify the user’s credentials on the WCF client proxy.  
   
 ### To configure a WCF service to authenticate using Windows domain username and password.  
   
-1.  Create an instance of the <<!--zz xref:System.ServiceModel.WsHttpBinding --> `xref:System.ServiceModel.WsHttpBinding`>, set the security mode of the binding to `SecurityMode.Message`, set the `ClientCredentialType` of the binding to `MessageCredentialType.UserName`, and add a service endpoint using the configured binding to the service host as shown in the following code:  
+1.  Create an instance of the `<xref:System.ServiceModel.WSHttpBinding%2A>`, set the security mode of the binding to `SecurityMode.Message`, set the `ClientCredentialType` of the binding to `MessageCredentialType.UserName`, and add a service endpoint using the configured binding to the service host as shown in the following code:
```
Comment (JRAlexander) [inline docs/framework/wcf/feature-details/how-to-authenticate-with-a-user-name-and-password.md]: Updated the xref here as well.

File: docs/framework/wcf/feature-details/how-to-authenticate-with-a-user-name-and-password.md [hunk 3]
```diff
@@ -90,7 +90,7 @@ This topic demonstrates how to enable a [!INCLUDE[indigo1](../../../../includes/
     ```  
   
 ## See Also  
- <<!--zz xref:System.ServiceModel.WsHttpBinding --> `xref:System.ServiceModel.WsHttpBinding`>  
+ <xref:System.ServiceModel.WSHttpBinding%2A>
```
Comment (JRAlexander) [inline docs/framework/wcf/feature-details/how-to-authenticate-with-a-user-name-and-password.md]: And here

**Review by JRAlexander — APPROVED** [review]
Thanks @wstaelens for your contribution! This was a good catch. I updated the xref and as soon as I verify the build I will get this merged. It should show up on the site in 1 or 2 days. Nice work!

**Comment by mairaw** [comment]
Overload links don’t work for classes and the link inside backticks so it won’t be interpreted as a link.

**Comment by mairaw** [comment]
Looks good now! It's related to #3682.

**Comment by JRAlexander** [comment]
Thanks, @mairaw, for the clarification!

**Comment by mairaw** [comment]
There was one more link left. My last change triggered the CLA though. @wstaelens do you mind signing the license agreement? Otherwise, I'll just remove my last commit.

**Comment by JRAlexander** [comment]
@wstaelens Can you click on the sign link above? I'd really like to get this merged. Thanks!

**Comment by mairaw** [comment]
Let me try to remove the last commit and see if we get rid of the signing requirement.

**Comment by wstaelens** [comment]
Done

**Comment by mairaw** [comment]
Thank you @wstaelens! Closed and reopened the PR because the build was stuck.

**Pull request merged**